tests/integration/example/example_test.go

```
@@ -155,48 +155,48 @@ func Example_oneModule() {
+	//result, err := integrationApp.RunMsg(&authtypes.MsgUpdateParams{
+	//	Authority: authority,
+	//	Params:    params,
+	//},
+	//	// this allows to the begin and end blocker of the module before and after the message
+	//	integration.WithAutomaticFinalizeBlock(),
+	//	// this allows to commit the state after the message
+	//	integration.WithAutomaticCommit(),
+	//)
+	//if err != nil {
+	//	panic(err)
+	//}
 
 	// verify that the begin and end blocker were called
 	// NOTE: in this example, we are testing auth, which doesn't have any begin or end blocker
 	// so verifying the block height is enough
-	if integrationApp.LastBlockHeight() != 2 {
-		panic(fmt.Errorf("expected block height to be 2, got %d", integrationApp.LastBlockHeight()))
-	}
-
-	// in this example the result is an empty response, a nil check is enough
-	// in other cases, it is recommended to check the result value.
-	if result == nil {
-		panic(fmt.Errorf("unexpected nil result"))
-	}
+	//if integrationApp.LastBlockHeight() != 2 {
+	//	panic(fmt.Errorf("expected block height to be 2, got %d", integrationApp.LastBlockHeight()))
+	//}
+	//
+	//// in this example the result is an empty response, a nil check is enough
+	//// in other cases, it is recommended to check the result value.
+	//if result == nil {
+	//	panic(fmt.Errorf("unexpected nil result"))
+	//}
 
 	// we now check the result
-	resp := authtypes.MsgUpdateParamsResponse{}
-	err = encodingCfg.Codec.Unmarshal(result.Value, &resp)
-	if err != nil {
-		panic(err)
-	}
+	//resp := authtypes.MsgUpdateParamsResponse{}
+	//err = encodingCfg.Codec.Unmarshal(result.Value, &resp)
+	//if err != nil {
+	//	panic(err)
+	//}
 
 	sdkCtx := sdk.UnwrapSDKContext(integrationApp.Context())
 
 	// we should also check the state of the application
 	got := accountKeeper.GetParams(sdkCtx)
-	if diff := cmp.Diff(got, params); diff != "" {
+	if diff := cmp.Diff(got, authtypes.DefaultParams()); diff != "" {
 		panic(diff)
```

Panics here, because all values are zero rather than correct defaults